### PR TITLE
add synopsis to architecture, transactions, accounts

### PR DESCRIPTION
**Pull request by rob-Hitchens**

### CODE REVIEW

File: academy/2-main-concepts/accounts.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ tag: deep-dive
 
 # Accounts
 
+<HighlightBox type="synopsis">
+
+Cryptographic signatures are indispensible components of blockchain technology. What are they, exactly, and how are they used in Cosmos applications? Public Key Cryptography has been used for decades. The math works. But what exactly does it do and how does it do it? What is meant by an "account" and how are the implemented in Cosmos?
```
Comment (xavierlepretre) [inline academy/2-main-concepts/accounts.md:12]: 1. typo: indispensable?
2. typo: how are **they** implemented

File: academy/2-main-concepts/transactions.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ tag: deep-dive
 
 # Transactions
 
+<HighlightBox type="synopsis">
+
+Transactions are a core concept in blockchain technology. After all, nothing  happens except through a successful transaction. Discover the components and life-cycle of transactions in Cosmos blockchains - how they are created by Cosmos applications, broadcast, confirmed and processed.
```
Comment (xavierlepretre) [inline academy/2-main-concepts/transactions.md:12]: I think the style guide is to have a comma in "confirmed, and processed."

**Review by xavierlepretre — APPROVED** [review]

File: academy/2-main-concepts/architecture.md [hunk 3]
```diff
@@ -5,9 +5,15 @@ description: ABCI, Tendermint, and state machines
...
 
-# A Blockchain App Architecture
+# Architecture
 
-In this section, let's take a closer look at the application architecture underlying blockchains built with the Cosmos SDK. You will look into Tendermint, consensus in Cosmos, the Application Blockchain Interface, the Cosmos SDK, and state machines. This section will give you a better feeling for the application architecture in Cosmos.
+<HighlightBox type="synopsis">
+
+The Cosmos SDK offers developers next-generation technology for decentralized applications. Discover how the Cosmos architecture decisively solves the most familiar challenges and limitations of blockchain platforms.
+
+This section unfolds the architecture of a blockchain application starting with low-level concerns such as consensus and upgradeability. Discover how application concerns are layered over a highly-efficient, general-purpose foundation that is equally applicable to public, private, and consortium blockchains. Then take a closer look at how to start creating a customized blockchain using the Cosmos SDK, superior to previous attempts creating general-purpose blockchains to run decentralized applications (dApps).
```
Comment (nooomski) [inline academy/2-main-concepts/architecture.md:14]: We don't need to boast so much, let the tech speak for itself :)

```suggestion
The Cosmos SDK offers developers next-generation technology for decentralized applications. Discover how the Cosmos architecture solves the most familiar challenges and limitations of blockchain platforms.

This section unfolds the architecture of a blockchain application, starting with low-level concerns such as consensus and upgradeability. Discover how application concerns are layered over a highly-efficient, general-purpose foundation that is equally applicable to public, private, and consortium blockchains. Then take a closer look at how to start creating a customized blockchain using the Cosmos SDK, and learn about the benefits of this method compared to creating general-purpose blockchains to run decentralized applications (dApps).
```

**Review by nooomski — COMMENTED** [review]
Thanks!

**Comment by coldice** [comment]
@xavierlepretre also this one - do we want to retain any of these changes?

**Comment by xavierlepretre** [comment]
> @xavierlepretre also this one - do we want to retain any of these changes?

We no longer have the synopsis box, and they were revised in a more recent round, so I say no need these changes.

**Pull request closed without merging**